BitwardenShared/Core/Auth/Repositories/AuthRepository.swift

```diff
@@ -590,7 +590,8 @@ extension DefaultAuthRepository: AuthRepository {
     }
 
     func passwordStrength(email: String, password: String) async throws -> UInt8 {
-        try await clientService.auth().passwordStrength(password: password, email: email, additionalInputs: [])
+        try await clientService.auth(isPreAuth: true)
+            .passwordStrength(password: password, email: email, additionalInputs: [])
     }
```

Comment:
🤔 Is there a potential drawback if someone in the future uses this `passwordStrength` method while the vault is unlocked? I mean someone could just see the protocol definition and miss this implementation detail. Maybe just a comment on the protocol definition would be enough, if needed.

Reply:
Maybe! I could expose it as a parameter here? I was thinking that `passwordStrength` probably isn't specific to the user, so maybe it wouldn't matter, but that could change.